src/components/LoginSignUp/AuthenticationService.ts

```diff
@@ -236,7 +231,7 @@ const getJWT = async ( allowAnonymousJWT = false ): Promise<string | null> => {
     try {
       response = await api.get<{api_token: string}>( "/users/api_token.json" );
     } catch ( getUsersApiTokenError ) {
-      logger.error( "Failed to fetch JWT: ", getUsersApiTokenError );
+      if ( !getUsersApiTokenError ) { return null; }
```

Comment:
`getUsersApiTokenError` cannot be falsy. I don't see this error in the logs so I don't think it's a liability, so I would err on the side of logging it. If you disagree, I don't think we need this try/catch.